yateto/generator.py

```diff
@@ -341,7 +367,7 @@ def unit_test_body(cpp, testFramework):
     print('Calling external code generators...')
     with Cpp(fRoutines.h) as header:
       with header.HeaderGuard(self._headerGuardName(namespace, self.ROUTINES_FILE_NAME)):
-        cache.generate(header, fRoutines.cpp)
+        cache.generate(header, fRoutines.cpp, fGpulikeRoutines.cpp)
```

Comment:
Do we want to generate the gpulike routines always? Even when not using a gpu?

Reply:
We generate **subroutines.cpp** and **subroutines.h** even if we don't use libsxmm. In this case, the files are just empty. I followed the same idea. If the user don't use GPU then **fGpulikeRoutines.cpp** is going to be empty. I cannot see  a problem here